hypershift-operator/metrics.go

```diff
@@ -485,6 +504,15 @@ func (m *hypershiftMetrics) observeNodePools(ctx context.Context, nodePools *hyp
 	return nil
 }
 
+func (m *hypershiftMetrics) observeOperator(ctx context.Context, hypPod corev1.Pod) error {
+	hypVersion := version.GetRevision()
```

Comment:
nit: for naming can we use `hypershiftVersion` or just `version`? same for all variables consistently